packages/twenty-website/src/github-synch/contributors/fetch-assignable-users.tsx

```diff
@@ -1,6 +1,6 @@
 import { graphql } from '@octokit/graphql';
 
-import { Repository } from '@/app/contributors/api/types';
+import { Repository } from '@/github-synch/contributors/types';
```

Comment:
synch is an unusual name, we usually use sync or synchronization (sync is better imo)